bin/stripdata.js

```diff
@@ -121,6 +123,21 @@ function splitDeviceId(deviceId) {
 	return retlist;
 }
 
+function removeIDsAndPayload(data) {
+	if (program.stripAll) {
+		if (data.payload)
+			delete data.payload;
+		if (data.id)
+			delete data.id;
+		if (data.guid)
+			delete data.guid;
+		if (data.bolus)
+			delete data.bolus;
+		if (data.suppressed)
+			delete data.suppressed;
```

Comment:
Deleting data.bolus deletes the bolusID in the wizard type. I'll work on fixing the suppressed data now.